code/_onclick/ai_onclick.dm

```diff
@@ -177,6 +177,11 @@
 // APC
 
 /obj/machinery/power/apc/AICtrlClick(mob/living/user) // turns off/on APCs.
+	if(aidisabled)
+		to_chat(user, "<span class='warning'>The AI wire is cut - Cannot toggle breaker.</span>")
+		return
+	else if (stat & BROKEN)
```

Comment:
This else here is not needed, since the previous statement returns. Basically just a small syntax change.
```suggestion
	if (stat & BROKEN)
```